app/models/metric/chargeback_helper.rb

```diff
@@ -22,7 +22,15 @@ def tag_list_with_prefix
     if resource.kind_of?(Container)
       state = resource.vim_performance_state_for_ts(timestamp.to_s)
       image_tag_name = "#{state.image_tag_names}|" if state
-      labels = resource.try(:container_image).try(:docker_labels).try(:collect) { |l| "container_image/label/managed/#{l.name}/#{l.value}" }
+      labels = resource.try(:container_image).try(:docker_labels).try(:collect) do |l|
+        escaped_name = AssignmentMixin.escape(l.name)
+        escaped_value = AssignmentMixin.escape(l.value)
+        [
+          # The assignments in tags can appear the old way as they are, or in base64
+          "container_image/label/managed/#{l.name}/#{l.value}",
+          "container_image/label/managed/#{escaped_name}/#{escaped_value}"
+        ]
```

Comment:
@cben Im all for it but im not sure where that helper would go since its a helper we sue for a mixin and a helper :sweat: